scripts/demo/demo_security_barrier_camera.sh

```diff
@@ -174,26 +174,26 @@ fi
 OS_PATH=$(uname -m)
 NUM_THREADS="-j2"
 
-if [ $OS_PATH == "x86_64" ]; then
+if [ "$OS_PATH" == "x86_64" ]; then
   OS_PATH="intel64"
   NUM_THREADS="-j8"
 fi
 
 build_dir="$HOME/inference_engine_demos_build"
-if [ -e $build_dir/CMakeCache.txt ]; then
-	rm -rf $build_dir/CMakeCache.txt
+if [ -e "$build_dir/CMakeCache.txt" ]; then
+  rm -rf "$build_dir/CMakeCache.txt"
```

Comment:
Why do we use the difference number of spaces instead of tab in different scripts?

Here we use 2 spaces but in other scripts we also use 4 spaces.

Reply:
fixed